src/WebJobs.Script/Diagnostics/MetricEventNames.cs

```diff
@@ -8,7 +8,8 @@ public static class MetricEventNames
         // host level events
         public const string ApplicationStartLatency = "host.application.start";
         public const string HostStartupLatency = "host.startup.latency";
-        public const string ApplicationInsightsEnabled = "host.applicationinsights";
+        public const string ApplicationInsightsEnabled = "host.applicationinsights.enabled";
+        public const string ApplicationInsightsDisabled = "host.applicationinsights.disabled";
```

Comment:
When looking at metrics to determine if someone had app insights enabled, I realized it was tough to see if they had ever disabled it (i.e they had it on 2 days ago, but turned it off yesterday). By logging both states, we can look for the last appinsights event to see if it's on or off.